site/docs/v1/tech/installation-guide/kickstart.adoc

```diff
@@ -430,10 +464,22 @@ And the usage in the kickstart file:
 
 === Reference
 
-The following environment variables  are built in:
+The following environment variables are built in:
 
-* `FUSIONAUTH_APPLICATION_ID` - The Id of the FusionAuth application
-* `FUSIONAUTH_TENANT_ID` - The Id of the default Tenant which is where the FusionAuth application resides
+* `FUSIONAUTH_APPLICATION_ID` - The Id of the FusionAuth application.
+* `FUSIONAUTH_TENANT_ID` - The Id of the default Tenant. This is always the Tenant which contains the FusionAuth application.
+* `FUSIONAUTH_FORM[adminUser]_ID` - The Id of the default admin user editing form.
```

Comment:
You can remove `editing` - this is for add and edit. 

You could leave as is, or extend it as:

> The Id of the default admin user form used for adding and editing users in the Admin UI.

Reply:
Thanks!